docs/shared-content/scripts/replace-certificate-scripts.include.md

```diff
@@ -0,0 +1,97 @@
+```powershell
+# Define working variables
+$octopusURL = "https://youroctourl"
+$octopusAPIKey = "API-YOURAPIKEY"
+$header = @{ "X-Octopus-ApiKey" = $octopusAPIKey }
+$space = "Spaces-1"
+
+# Certificate details
+$certificateName = "MyCertificate"
+$certificateFilePath = "path\to\pfx-file.pfx"
+$certificatePfxPassword = "PFX-file-password"
+
+# Convert PFX file to base64
+$certificateContent = [Convert]::ToBase64String((Get-Content -Path $certificateFilePath -Encoding Byte))
+
+try
+{
+    # Get existing certificate
+    $certificate = (Invoke-RestMethod -Method Get -Uri "$octopusURL/api/$space/certificates/all" -Headers $header) | Where-Object {$_.Name -eq $certificateName}
```

Comment:
What happens if there are multiple certificates which already exists with that name? The search doesnt cover ones already replaced or archived:

You can alter the query above to:

```powershell
| Where-Object {
           $_.Name -eq $certificateName
           $null -eq $_.ReplacedBy -and
           $null -eq $_.Archived
}
```

In addition, you could either pipe that to `Select-Object -First 1` OR return a warning in case there are multiple matches?

Reply:
Good catch!  Changed to include the archived portion as I don't think the other part is relevant.